specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13936,6 +16885,10 @@
         }
       }
     },
+    "Object": {
```

Comment:
Let me work with @naveedaz to see if I can author this so the tooling which spews the Swagger doesn't emit this.